app/components/primer/underline_nav_component.rb

```diff
@@ -100,31 +117,38 @@ class UnderlineNavComponent < Primer::Component
     #
     # @param with_panel [Boolean] Whether the TabNav should navigate through pages or panels.
     # @param align [Symbol] <%= one_of(Primer::UnderlineNavComponent::ALIGN_OPTIONS) %> - Defaults to <%= Primer::UnderlineNavComponent::ALIGN_DEFAULT %>
+    # @param body_arguments [Hash] <%= link_to_system_arguments_docs %> for the body wrapper.
     # @param system_arguments [Hash] <%= link_to_system_arguments_docs %>
-    def initialize(with_panel: false, align: ALIGN_DEFAULT, body_classes: "", **system_arguments)
+    def initialize(with_panel: false, align: ALIGN_DEFAULT, body_arguments: { tag: BODY_TAG_DEFAULT }, **system_arguments)
```

Comment:
🤔 interesting. I wonder if this API could prove tricky to use. For example, if someone passes in `body_arguments`, they need to be sure to pass in a `tag` key. What do you think about having `body_classes` and `body_tag`? Then again, this is starting to feel like a slot...